demo/src/examples/example.module.css

```
@@ -109,3 +109,43 @@
+  align-items: baseline;
+  gap: 0;
+}
+
+.inlineSuggestionPrefix {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  border: 0;
+  margin: -1px;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)